figures/S5.supplemental_figure5/scripts/S5.r

```diff
@@ -0,0 +1,1448 @@
+suppressPackageStartupMessages(suppressWarnings(library(ggplot2))) # plotting
+suppressPackageStartupMessages(suppressWarnings(library(dplyr))) # data manipulation
+suppressPackageStartupMessages(suppressWarnings(library(patchwork))) # figure composition
+suppressPackageStartupMessages(suppressWarnings(library(cowplot))) # figure composition
+suppressPackageStartupMessages(suppressWarnings(library(RcppTOML))) # parsing config file
+suppressPackageStartupMessages(suppressWarnings(library(pheatmap))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(lattice))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(RColorBrewer))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(gplots))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(ComplexHeatmap))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(ggplotify))) # grob
+suppressPackageStartupMessages(suppressWarnings(library(viridis))) # color
+suppressPackageStartupMessages(suppressWarnings(library(platetools))) # make plate plot
+suppressPackageStartupMessages(suppressWarnings(library(circlize)))
+suppressPackageStartupMessages(suppressWarnings(library(reshape2))) # data manipulation
+suppressPackageStartupMessages(suppressWarnings(library(stringr))) # string manipulation
+suppressPackageStartupMessages(suppressWarnings(library(purrr))) # data manipulation
```

Comment:
Consider using for loop to iterate over a vector with package names and load with suppressed messages.

Reply:
ooo good idea! Implemented!